shapes/contacts/person.shex

```diff
@@ -0,0 +1,47 @@
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX dc: <https://dublincore.org/specifications/dublin-core/dcmi-terms/#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+# We should store the preferred WebId for each person, since sameAs can be multiple WebIds
+
+vcard:Individual {
+    vcard:fn xsd:string {1} ; 
+    vcard:inAddressBook vcard:AddressBook + ;
+    owl:sameAs vcard:Individual * ; # WebIds
+    vcard:hasUID xsd:string ? ;
+    vcard:hasName xsd:string ? ; # structured name 
+    vcard:hasPhoto dc:Image * ; 
+    vcard:hasRelated vcard:RelatedType * ; 
+    vcard:url rdf:Resource * ; # need to describe the type of url to be able to identify WebIds 
+    vcard:hasAddress vcard:Address * ; 
+    vcard:bday xsd:date ? ; 
+    vcard:hasEmail vcard:Email * ; 
+    vcard:hasTelephone vcard:Tel * ; 
+    vcard:organization-name xsd:string ? ; 
+    vcard:role xsd:string ? ; 
+    vcard:title xsd:string ? ; 
+    vcard:note xsd:string ? ;
+}
+
+vcard:Address {
+    vcard:street-address xsd:string {1} ;
+    vcard:locality xsd:string {1} ; 
+    vcard:postal-code xsd:string {1} ;
+    vcard:region xsd:string {1} ;
+    vcard:country-name xsd:string {1} ;
+}
+
+vcard:Email {
+    vcard:value xsd:string {1}; # should have a `mailto` prefix
+}
+
+vcard:Tel {
```

Comment:
Similar comments as for email - type, regex, string lengths.

Refer to https://solidsdk.inrupt.net/sdk/userprofile.shex:

```<#UserProfileTelephone> {
    a [vcard:Dom vcard:Home vcard:ISDN vcard:Internet vcard:Intl vcard:Label vcard:Parcel vcard:Postal vcard:Pref vcard:Work vcard:X400] {1}
      // rdfs:label "Type"@en
      // rdfs:label "Tipo"@es ;
    vcard:value xsd:string /^\+?[0-9]+[0-9-]*[0-9]$/ {1}
      // rdfs:label "Phone"@en 
      // rdfs:label "TelÃ©fono"@es
      // :prefix "tel:" ;
    }
```